source/kraken/tests/realtime_test.cpp

```diff
@@ -70,6 +70,7 @@ make_cancellation_message(const std::string& vj_uri, const std::string& date) {
     trip->set_schedule_relationship(transit_realtime::TripDescriptor_ScheduleRelationship_CANCELED);
```

Comment:
note to self: this method should probably just be sugar around `make_delay_message` once renamed.